demos/checkboxradio/icons.html

```diff
@@ -0,0 +1,53 @@
+<!doctype html>
+<html lang="en">
+<head>
+	<meta charset="utf-8">
+	<title>jQuery UI Checkboxradio - Icon functionality</title>
+	<link rel="stylesheet" href="../../themes/base/all.css">
+	<script src="../../external/jquery/jquery.js"></script>
+	<script src="../../ui/core.js"></script>
+	<script src="../../ui/widget.js"></script>
+	<script src="../../ui/checkboxradio.js"></script>
+	<link rel="stylesheet" href="../demos.css">
+	<script>
+	$(function() {
+		$( "input" ).checkboxradio({
+			icon: true
+		});
+	});
+	</script>
+</head>
+<body>
+<div class="widget">
+	<h1>Checkbox and radio button widgets</h1>
+	<h2>Checkbox</h2>
+
+	<fieldset>
+		<legend>Rating</legend>
+		<label for="checkbox-1">2 Star</label>
+		<input type="checkbox" name="checkbox-1" id="checkbox-1">
+		<label for="checkbox-2">3 Star</label>
+		<input type="checkbox" name="checkbox-2" id="checkbox-2">
+		<label for="checkbox-3">4 Star</label>
+		<input type="checkbox" name="checkbox-3" id="checkbox-3">
+		<label for="checkbox-4">5 Star</label>
+		<input type="checkbox" name="checkbox-4" id="checkbox-4">
+	</fieldset>
+	<h2>Radio Group</h2>
+	<fieldset>
+		<legend>Location: </legend>
+		<label for="radio-1">New York</label>
+		<input type="radio" name="radio-1" id="radio-1">
+		<label>Paris
+			<input type="radio" name="radio-1" id="radio-2">
+		</label>
+		<input type="radio" name="radio-1" id="radio-3">
+		<label for="radio-3">London</label>
+	</fieldset>
+</div>
+
+<div class="demo-description">
+<p>Examples of the markup that can be used with checkboxs and radio buttons.</p>
```

Comment:
Also update this text then, add something like ", here showing both without icons."